Gaijinizer patch/CommonEvents_180.txt

```diff
@@ -33,16 +33,21 @@ Resist, resist♪#Strate
 
 こんな状況でも話しかけてくれるなんて、
 随分と余裕があるのですね。
-> CONTEXT: CommonEvents_180/62/Dialogue/yuuma < UNTRANSLATED
+> CONTEXT: CommonEvents_180/62/Dialogue/yuuma
+......Fufu.
 
+Even in such a state, you can still afford to
+talk to me.#Strate
 > END STRING
 
 > BEGIN STRING
 あなたが望んでいることはおおかた分かりますが……
 
 残念ながら、それは私が望むものとは違いますので…
-> CONTEXT: CommonEvents_180/67/Dialogue/yuuma < UNTRANSLATED
+> CONTEXT: CommonEvents_180/67/Dialogue/yuuma
+I know what you wish for the most......
 
+Unfortunately, I wish for something different...#StrateUnsure
 > END STRING
```

Comment:
Ron Note

Reply:
No JP issues. If you were to word it another way you could say, "I more-or-less know what it is you want, but unfortunately that's not what I want."

Reply:
Minor revision:
```
I know what you wish for the most......

Unfortunately, I want something different...
```